conf/modules.config

```diff
@@ -614,6 +614,7 @@ process {
     withName: 'MULTIQC' {
         ext.args   = { params.multiqc_title ? "--title \"$params.multiqc_title\"" : '' }
         publishDir = [
+            path: { "${params.outdir}/multiqc" },
```

Comment:
why the same line twice?

Reply:
Ah good catch :p that snuck in during merge conflict resolving